specification/datalake-analytics/resource-manager/readme.md

```diff
@@ -39,8 +39,6 @@ These settings apply only when `--tag=package-2016-11` is specified on the comma
 ``` yaml $(tag) == 'package-2016-11'
 input-file:
 - Microsoft.DataLakeAnalytics/2016-11-01/account.json
-- Microsoft.DataLakeAnalytics/2016-11-01/catalog.json

```

Comment:
Yes please, @sarangan12 please confirm with .NET and see whether this changes the .NET SDK or not.